src/js/utils/RuleExporter.js

```diff
@@ -26,7 +26,7 @@ export type JSONFormat = {
 
 type RulePropertyJSON = {
   attribute: ?string,
-  dateTimeFormat?: string,
+  format?: string,
```

Comment:
just out of curiosity, what triggered the change of name?

Reply:
This was a bug. This is the output format, and the SDK expects `format`. That explains why the preview never had the date =)

Reply:
I understand the SDK expects `format`, which was in fact what we were sending. I believe there was no need to change this name here, but it is not a show-stopper.